### ISSUE TITLE
RFC: Don't require plugins to have `main.js`

### ISSUE THREAD

**Issue by vitaliy-bobrov** [micbuffa/WebAudioPlugins#16]
The restriction to name plugin entry file name as `main.js` won't work for most cases, different environments would use different entries, ex. es5 / es2019  / ts versions.

**Comment by micbuffa** [comment]
Indeed. For the V2 of the SDK this would be a good addition to be able to specify the name of the main js file in the main.json descriptor.

**Comment by khoin** [comment]
I thought this is not the case?
In fact, it looks like the SDK, when calls `.load()` actually refers to the name of the class given in main.json. See here: https://github.com/micbuffa/WebAudioPlugins/blob/master/sdk/WebAudioSDK.js#L289
